### PR TITLE
[10.0][MIG] partner_password_reset

### PR DESCRIPTION
**Pull request by lasley**
Parner Password Reset
=====================

This module provides an action that allows users to send a password reset
email from the Partner view.

Forward port of #369:

* Rename manifest
* Update openerp references to odoo
* Bump version

### CODE REVIEW

File: partner_password_reset/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+None required
+
+Usage
+=====
+
+To use this module, simply select "Send password reset email" under the partner
+view.
+
+
```
Comment (elicoidal) [inline partner_password_reset/README.rst]: Can you add a link to the runbot?

**Review by elicoidal — APPROVED** [review]
Quite straightforward. 
Can you check Travis?

File: partner_password_reset/README.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Parner Password Reset
```
Comment (elicoidal) [inline partner_password_reset/README.rst]: "Partner"

File: partner_password_reset/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Partner User Password Reset",
```
Comment (elicoidal) [inline partner_password_reset/__manifest__.py]: Please sync with README title

**Review by hparfr — APPROVED** [review]
From an UI perspective, it's more consitant to put this action in the "Action" drop-down list.

**Comment by lasley** [comment]
@hparfr Agreed - that would also allow for multiple resets at the same time in the list view.

@tedsalmon [`key2="client_action_multi"`](https://github.com/odoo/odoo/blob/fc2e80cb4bcc450762c7ac5cb82a3e2d88062b38/addons/point_of_sale/wizard/pos_box.xml#L19)

**Comment by tedsalmon** [comment]
This required the functionality be moved out of an action and into a wizard, so I did just that and updated the tests. Whenever we have full sign off, I'll go ahead and squash the changes.

Thanks!

**Pull request merged**